### DISCUSSION TITLE
Different ratios of proofs found to number of plots on two harvesters.

### DISCUSSION THREAD

**Discussion by Jacek-ghub** [Chia-Network/chia-blockchain#8249] [Q&A] [answered]
I have a couple of harvesters that have about the same amount of plots (20% difference).  However, the ratios of proofs found to the number of farmed plots differs a lot.  First harvester has about 1:2 ratio of proofs found to # of plots, the other one has 1:10.  Those ratios are rather consistent, they basically don't fluctuate.

Both harvesters are sitting under the same full node, and processing the exactly same number of searches (as it should be).  Both have basically the same response averages (0.22 vs. 0.33) with zero responses over 5 sec.  

Also, I shifted about 50% of plots from one harvester to another, but before shift those ratios were even worst - 1:1 vs 1:10.  So, those plots moved from the 1:10 ratio harvester didn't change anything on that harvester, but dropped the ratio on the better one from 1:1 down to 1:2.

Is it possible that when (I mean by date of creation) those plots were created could be the reason?  The harvester with better ratio has all plots created with potentially the latest (v1.1.5) version of MadMax.  The other one has also all plots created with MM, but with older versions.  There are no plots created with the official ciha plotter.

Looking at the logs, all looks good, there are no ERRORs there.  At least, I don't know what eventually to look for.

**Comment by loppefaaret** [comment]
is it full proofs, or partials we talking ? how long is the sample periods ?

**Comment by Jacek-ghub** [comment]
All plots belong to a pool, so those are partials (I think).  The data comes from ChiaDog, and reports are being sent every 12 hours for the past week+

Although, I run PSChiaPlotter heat maps for over a month, and that is also what I see there (but cannot pull those ratios from those heat maps).

Both PSChiaPlotter and ChiaDog refer to those as proofs, though.

**Comment by keliew** [comment, marked as answer]
they all point to the same pool?

**Comment by Jacek-ghub** [comment]
(Yes, all plots belong to the same pool.)
I take it back, I have few test plots that belong to "test pools" - pools that I am comparing.  Let me move those plots off-line, and I will update the results later today.

**Comment by keliew** [comment]
1 in every 512 plots try to pass through the filter. How are you getting such high ratios of proofs found versus number of plots?

**Comment by Jacek-ghub** [comment]
If you read what I wrote before, I use ChiaDog and PSChiaPlotter heat maps.  They kind of agree (I can get exact number from ChiaDog, but only heat maps from PSChiaPlotter -  but those heat maps show the same pattern).

Again, as mentioned, I had few plots belonging to two extra pools, and I removed them already, so we should pause those exchanges till I get the new numbers / heat maps.

**Comment by Jacek-ghub** [comment]
@keliew Thank you for that pointer about different pools.  That was it.

Right now, those two main harvesters have about the same ratios (0.06 vs. 0.05).  So, we can close this threads.  The heat maps for those harvesters also agree.

The interesting part is that the plotter that holds that handful of plots belonging to two different pools has 1.68 ratio.  Also, the number of proofs for those testing plots is few times higher than on either of those main harvesters (even though there are only few of those).  I don't know how to explain that, but looking at pool stats, those two pools perform about the same, and as expected.

Although, I would like to see stats per harvester in the Chia UI, where we could see those numbers, so there would be no need to fish through those logs or use third party software that more or less is just dividing numbers.

Again, thank you @keliew!

**Reply by keliew** [reply]
> Although, I would like to see stats per harvester in the Chia UI, where we could see those numbers, so there would be no need to fish through those logs or use third party software that more or less is just dividing numbers.

That'd be very useful. And I also noticed that not all pools have the same mechanism for partials and points, so the result may be OK on Chia, but not on the pool's dashboard, vice versa.

It's rather annoying...

**Comment by Jacek-ghub** [comment]
> not all pools have the same mechanism for partials and points

That basically makes those pool reports worthless.

That is the reason that I keep around those few extra plots, and run them against two different pools.  This way, I don't care who reports what, as after some time cumulative results should match (and they don't - small sample, but ...).

So, I fully agree that those mechanisms vary.  The sad part is that it may incentivize some pool owners to keep zero fees for longer, but do some creative accounting on how they distribute earnings.  Again, rather poor design on the official Chia that they don't have a back propagating enforcement from those pools (so one can compare what the official UI has to say about those earnings to what a given pool reports).  They could easily aggregate those pool reports, and compare what they delivered to those pools.
